docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,402 @@
+
+You can see our Ropsten Kube configurations https://github.com/keep-network/keep-core/tree/master/infrastructure/kube/keep-test[here]
+
+== Logging
+
+Below are some of the key things to look out for to make sure you're booted and connected to the
+network:
+
+=== Configurable Values
+
+```
+LOG_LEVEL=DEBUG
+IPFS_LOGGING_FMT=nocolor
+GOLOG_FILE=/var/log/keep/keep.log
+GOLOG_TRACING_FILE=/var/log/keep/trace.json
+```
+
+=== Startup
+```
+▓▓▌ ▓▓ ▐▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▌▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓    ▓▓▓▓▓▓▓▀    ▐▓▓▓▓▓▓    ▐▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▄▄▓▓▓▓▓▓▓▀      ▐▓▓▓▓▓▓▄▄▄▄         ▓▓▓▓▓▓▄▄▄▄         ▐▓▓▓▓▓▌   ▐▓▓▓▓▓▓
+  ▓▓▓▓▓▓▓▓▓▓▓▓▓▀        ▐▓▓▓▓▓▓▓▓▓▓         ▓▓▓▓▓▓▓▓▓▓▌        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
+  ▓▓▓▓▓▓▀▀▓▓▓▓▓▓▄       ▐▓▓▓▓▓▓▀▀▀▀         ▓▓▓▓▓▓▀▀▀▀         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▀
+  ▓▓▓▓▓▓   ▀▓▓▓▓▓▓▄     ▐▓▓▓▓▓▓     ▓▓▓▓▓   ▓▓▓▓▓▓     ▓▓▓▓▓   ▐▓▓▓▓▓▌
+▓▓▓▓▓▓▓▓▓▓ █▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓ ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓ ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓  ▓▓▓▓▓▓▓▓▓▓
+
+Trust math, not hardware.
+
+-----------------------------------------------------------------------------------------------
+| Keep Random Beacon Node                                                                     |
+|                                                                                             |
+| Port: 3919                                                                                  |
+| IPs : /ip4/127.0.0.1/tcp/3919/ipfs/16Uiu2HAmCcfVpHwfBKNFbQuhvGuFXHVLQ65gB4sJm7HyrcZuLttH    |
+|       /ip4/10.102.0.112/tcp/3919/ipfs/16Uiu2HAmCcfVpHwfBKNFbQuhvGuFXHVLQ65gB4sJm7HyrcZuLttH |
+-----------------------------------------------------------------------------------------------
+```
+
+**Bonus**: If you want to share your LibP2P address with others you can get it from the startup log.  When sharing remember to substitute the `/ipv4/` address with the public facing IP of your client if you're running on a private machine, or replace the entire `/ipv4/` segment with a DNS entry if you're using a hostname.
+
+=== Peer Connections
+
+```
+21:19:47.129 DEBUG keep-net-w: connected to [1] peers:[16Uiu2HAm3eJtyFKAttzJ85NLMromHuRg4yyum3CREMf6CHBBV6KY]
+```
+
+== ETH Networks
+
+=== Mainnet
+
+==== Boostrap Peers
+
+==== Contracts
+
+[%header,cols=2*]
+|===
+|Token
+|
+
+|KeepToken
+|''
+
+|TokenStaking
+|''
+
+|TokenGrant
+|''
+|===
+
+[%header,cols=2*]
+|===
+|Group
+|
+
+|Groups
+|''
+
+|GroupSelection
+|''
+|===
+
+[%header,cols=2*]
+|===
+|RandomBeacon
+|
+
+|DKGResultVerification
+|''
+
+|KeepRandomBeaconServiceImplV1
+|''
+
+|KeepRandomBeaconService
+|''
+
+|KeepRandomBeaconOperator
+|''
+|===
+
+=== Testnet
+
+Keep uses the Ethereum Ropsten Testnet.
+
+==== Bootstrap Peers
+
+[small]*`"/dns4/bootstrap-0.test.keep.network/tcp/3919/ipfs/16Uiu2HAmCcfVpHwfBKNFbQuhvGuFXHVLQ65gB4sJm7HyrcZuLttH"`*
```

Comment:
Not sure what this `[small]` is meant to be but it's not clear to me that it's doing something expected <_<

Reply:
It literally just shrinks the text...it was so I could get each peer on a single line.

Reply:
They are not on single lines ->

<img width="1024" alt="Screen Shot 2020-04-09 at 11 54 18" src="https://user-images.githubusercontent.com/8245/78914743-d9eb2b00-7a58-11ea-8d45-5a1985d89c8b.png">

https://docs.keep.network/sthompson22/docs/run-keep/run-random-beacon.html#_bootstrap_peers

Reply:
Might need to do something like:

```
[small]
====
"/dns4/..."
"/dns4/..."
…
====
```

Reply:
Weird - looks ok in my asciidoctor preview.  Pushing an update.

Reply:
Lol well now they're all bold but it'll do.